src/js/messaging/containers/Thread.jsx

```diff
@@ -328,9 +345,17 @@ export class Thread extends React.Component {
       return <p>Sorry, this message does not exist.</p>;
     }
 
+    // check if any senderIds are valid recipients
+    const disabled = intersection(
+      recipients.map(e => e.value),
```

Comment:
Does this need to check that the recipients are loaded first? I see a check https://github.com/department-of-veterans-affairs/vets-website/pull/4846/files#diff-a5c007b1e7ed1bbf23e95311462babb4R321 but also requires the isNewMessage to be true, so it does seem possible to get to this statement if !isNewMessage && loading.recipients